web-ui/src/main/resources/catalog/views/default/directives/partials/mdactionmenu.html

```diff
@@ -154,6 +154,17 @@
           <span data-translate="">enableWorkflow</span>
         </a>
       </li>
+      <!--metadata workflow assistance external link(s)-->
+      <li role="menuitem"
+          data-ng-if="isMdWorkflowEnable && md.isWorkflowEnabled()
+          && isMdWorkflowAssistEnable && user.canEditRecord(md)" data-ng-repeat="app in workFlowApps">
+        <a href=""
+           data-ng-href="{{app.appUrl.replace('{uuid}', md.getUuid()).replace('{lang}', lang).replace('{isoLang}', iso2Lang)}}"
+           target="_blank" rel="noopener">
+          <span class="fa fa-fw fa-user"></span>&nbsp;
```

Comment:
I think you should change fa-user to fa-external-link so you get something like the following 
![image](https://user-images.githubusercontent.com/1868233/158801333-c3b5bdec-50fa-4211-85a5-9ba7886a435b.png)